src/documentation/components/ComponentExample.vue

```diff
@@ -22,7 +22,10 @@
     </h3>
     <slot name="example-description"></slot>
     <div class="example -mb--3">
-      <div :class="padding || '-p--3'">
+      <div
+        :class="padding || '-p--3'"
+        :style="'backgroundColor:' + backgroundColor || '#fff'"
```

Comment:
Style is not going to scale well. I would suggest include a prop called `additionalClasses`. It would allow doc examples scale in a better way.